tests/ui/attributes/unsafe/extraneous-unsafe-attributes.rs

```diff
@@ -27,4 +27,8 @@ mod inner {
 #[unsafe(used)] //~ ERROR: is not an unsafe attribute
 static FOO: usize = 0;
 
-fn main() {}
+fn main() {
+    let _a = cfg!(unsafe(foo));
+    //~^ ERROR: expected identifier, found keyword `unsafe`
+    //~^^ ERROR: invalid predicate `r#unsafe`
```

Comment:
Nit: `//~| ERROR:`

Reply:
When there is 2 or more annotations I prefer them to be one below the other at the same indentation, makes it easier to read.